src/docs-style-guide.rst

```diff
@@ -714,6 +938,34 @@ XForms and XLSForm
 - an Xform (when referring to a single form)
 - XLSForm
 
+.. code-block:: python
```

Comment:
This code block should probably be two code blocks --- one for the ODK stuff, and one for the XForms/XLSForm stuff.

Also, `Odk` should should get caught.